node_modules/.bin/acorn

```diff
@@ -0,0 +1 @@
+../acorn/bin/acorn
```

Comment:
- [ ] great job setting up the project 👍  however the node_modules should be added to the git ignore file, please remove the folder and add it to the .gitignore file.

Reply:
Thanks, totally forgot about it.